### PR TITLE
#992 Add GCC-4.1 by default

### PR DESCRIPTION
**Pull request by uilianries**
- Included gcc-4.1 in default settings.yml

Signed-off-by: Uilian Ries <uilianries@gmail.com>

### CODE REVIEW

File: conans/client/migrations.py [hunk 1]
```diff
@@ -44,7 +44,7 @@ def _make_migrations(self, old_version):
 arch: [x86, x86_64, ppc64le, armv6, armv7, armv7hf, armv8]
 compiler:
     gcc:
-        version: ["4.4", "4.5", "4.6", "4.7", "4.8", "4.9", "5.1", "5.2", "5.3", "5.4", "6.1", "6.2"]
```
Comment (memsharded) [inline conans/client/migrations.py]: The new setting doesn't have to be included here. This is a migration, will use the ``old_settings`` to know if the user changed something from the previous versions defaults.

Comment (uilianries) [inline conans/client/migrations.py]: Understood

**Comment by uilianries** [comment]
Thanks, guys!

**Comment by memsharded** [comment]
Implements https://github.com/conan-io/conan/issues/992

**Pull request merged**